src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1031,6 +1056,10 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
       .div(userCurrencyToUsdRate)
       .toFixed()
 
+    const slippageDecimalPercentage = bnOrZero(slippageFiatUserCurrency)
```

Comment:
`estimateAddThorchainLiquidityPosition` already returns this if we just want to set the value from the `useEffect` already setting `slippageFiatUserCurrency`. Then we can just react on that state var and not add to the `confirmedQuote` which isn't used by anything downstream at this time

Reply:
Agree there's no need to add to the `confirmedQuote`, at least not for now! 34bd63acf3